recipe/meta.yaml

```
@@ -25,17 +25,11 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python
-    - py4j ==0.10.9.7
-    # extras_require dependencies are kept here
-    # as runtime dependencies because previously
-    # were set as such
-    - numpy >=1.15,<2
-    - pandas >=1.0.5
-    - pyarrow >=4.0.0
-  run_constrained:
-    # We haven't previously included the "connect" extras, 
-    # so we can keep these in run_constrained.
+    - python >=3.8
+    - py4j ==0.10.9.9
+    - numpy >=1.21
+    - pandas >=2.0.0
+    - pyarrow >=11.0.0
     - grpcio >=1.56.0
     - grpcio-status >=1.56.0
     - googleapis-common-protos >=1.56.4
```

```suggestion
    - grpcio >=1.67.0
    - grpcio-status >=1.67.0
    - googleapis-common-protos >=1.65.0
```
Looks like these got bumped: https://inspector.pypi.io/project/pyspark/4.0.1/packages/ae/40/1414582f16c1d7b051c668c2e19c62d21a18bd181d944cb24f5ddbb2423f/pyspark-4.0.1.tar.gz/pyspark-4.0.1/setup.py#line.156